integration-tests/src/test/resources/playground/test-processor/src/main/kotlin/BuilderProcessor.kt

```diff
@@ -34,6 +34,16 @@ class BuilderProcessor : SymbolProcessor {
             val parent = function.parentDeclaration as KSClassDeclaration
             val packageName = parent.containingFile!!.packageName.asString()
             val className = "${parent.simpleName.asString()}Builder"
+
+            // For regression testing https://github.com/google/ksp/pull/467
+            codeGenerator.createNewFile(
+                Dependencies(true, function.containingFile!!),
+                "",
+                "META-INF/proguard/builder-${className}.pro"
```

Comment:
Changing this to `"META-INF/proguard/builder-${className}", "pro"` seems to fix the errors for me locally.

It shouldn't end up with NPE though. We'll keep working on error handling.

Reply:
Got it, confirmed that resolved it. Let me update the test now